beyond-essence/inventory/buttons.md

```diff
@@ -1,6 +1,99 @@
 ---
 layout: beyond-essence
+header_image: private/main.jpg
+title: Buttons
+background_image: index.jpg
 ---
 
-Buttons are basic elements in user interfaces.
-It is important to design an effective button that will make the user take action.
+Buttons are interactive elements and belong to the action controls.
+They allow the user to directly communicate with the user interface.
+
+## Use cases
+
+With buttons we allow the user to take an action upon click or touch.
+Buttons appear at various places throughout the UI, such as:
+
+* Cards
+* Dialogs
+* Settings layers
+* Top bar
+* Bottom bar
+* Themes
+* Editor
+* Empty states
+
+## Structure
+
+Buttons are characterized by a certain geometric shape or border.
+Their color clearly stands out from the surrounding color.
+The UI label appears on the button, and clearly explains what the button will do.
+In exeptional cases or where the situation requires it, buttons appear with an icon instead of a UI label.
+
+## Position
+
+Buttons are positioned where users can easily find them or expect to see them.
+The position of the buttons should be intuitive for the user and reflect a natural conversation between the user and the software.
+The order and position of buttons is especially important when they come in pairs, such as `Cancel` and `Save`.
+If the user has the choice between two buttons, one distinguishes between primary and secondary buttons.
+The primary button is the one that completes the user's task, whereas the secondary button has less importance and is rather counterproductive for task completion.
```

Comment:
We also have a default button which is always the gray 'Cancel' button. What about a type section which short describe (and with picture) for what this button is for?
Primary (green)
Secondary (outline)
Default (gray)
Danger (red)